src/ArtifactPaths.jl

```diff
@@ -0,0 +1,62 @@
+module ArtifactPaths
+
+using Artifacts
+import ..get_artifact_path
+
+export get_lookup_filename, get_input_filename
+
+"""
+    get_lookup_filename(optics_type::S, λ::S) where {S<: Symbol}
+
+This function generates the file names for lookup table files for a given optics type and 
+wavelength type. 
+`:gas`, `:cloud` and `:aerosol` optics types are supported. 
+`:lw` (longwave) and `:sw` (shortwave) wavelength types are supported.
+
+These artifacts are obtained from https://github.com/earth-system-radiation/rrtmgp-data package.
+"""
+function get_lookup_filename(optics_type::S, λ::S) where {S <: Symbol}
```

Comment:
```suggestion
function get_lookup_filename(optics_type::Symbol, λ::Symbol)
```
Symbol doesn't have subtypes and is not parameteric